### PR TITLE
Update stations info test

### PR DESCRIPTION
**Pull request by chiragrank**
Changes
* Update the tests
* Update function for better exception handling

### CODE REVIEW

File: noaastn/noaastn.py [hunk 1]
```diff
@@ -66,10 +75,10 @@ def get_stations_info(country="all"):
             if i <= skip_lines:
                 continue
             for i in range(len(col_index) - 1):
-                val = line[col_index[i]: col_index[i + 1]].strip()
+                val = line[col_index[i] : col_index[i + 1]].strip()
```
Comment (spentelow) [inline noaastn/noaastn.py]: Looks like this line is throwing an error on the build action.
```suggestion
                val = line[col_index[i]: col_index[i + 1]].strip()
```

File: noaastn/noaastn.py [hunk 2]
```diff
@@ -66,10 +75,10 @@ def get_stations_info(country="all"):
...
                 if len(val) > 0:
                     data[i].append(
-                        line[col_index[i]: col_index[i + 1]].strip()
+                        line[col_index[i] : col_index[i + 1]].strip()
```
Comment (spentelow) [inline noaastn/noaastn.py]: Looks like this line is throwing an error on the build action.
```suggestion
                        line[col_index[i]: col_index[i + 1]].strip()
```

**Review by spentelow — CHANGES_REQUESTED** [review]
There are a couple of spots where the git-build action is failing because the linter is unhappy with whitespaces.  I've removed the whitespaces and added them to your PR as suggestions - hopefully that works smoothly

**Comment by chiragrank** [comment]
@spentelow . You are right the linter seemed unhappy at those places. Running the linter locally, there were some more errors that seemed to be showing at my end. I fixed that and had already pushed the changes before looking at your message here. Hope that is okay

**Pull request merged**